apps/discovery_streams/lib/discovery-streams/event_handler.ex

```diff
@@ -24,36 +24,37 @@ defmodule DiscoveryStreams.EventHandler do
         data: %Dataset{technical: %{private: true}} = dataset,
         author: author
       }) do
-    dataset_update()
-    |> add_event_count(author, dataset.id)
+    add_event_count(dataset_update(), author, dataset.id)
 
+    # TODO: need to turn off source
```

Comment:
oh yeah, that'd be good

Reply:
Yea  i actually wrote that line of code below, but I havent tested it yet, so i've left the todo lol.  We used to just let topic manager handle all that as async tasks so there werent obvious tests to rewrite that i saw.